bpo-36543: Restore cElementTree and mark it for removal in 3.10 (#19921)
python/cpython: Doc/whatsnew/3.9.rst

  `parso`_.
  (Contributed by Carl Meyer in :issue:`40360`.)

* The :mod:`xml.etree.cElementTree` module has been deprecated since Python
  3.3 and is scheduled to be removed in Python 3.10.

>   (Contributed by Serhiy Storchaka and Christian Heemes in :issue:`36543`)

Reviewer: A typo in your own name? That's rare, I guess. ;-)

Author: You wouldn't believe me :+1:

  `parso`_.
  (Contributed by Carl Meyer in :issue:`40360`.)

* The :mod:`xml.etree.cElementTree` module has been deprecated since Python

>   3.3 and is scheduled to be removed in Python 3.10.

Reviewer 2: Use more direct wording:
* clearer
* similar to that in xml.etree.elementtree.rst
* fewer words = more readable

```suggestion
  3.3 and will be removed in Python 3.10.
```
